docs/topics/forms/descriptions/location-description.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Description location
+layout: default
+parent: Input and description
+nav_order: 4
+---
+
+# Place descriptions between label and form field
+
+{: .info }
+Note: this is best practice and not a WCAG requirement.
+
+Users read from the top down in a form. Place descriptions in a consistent location, preferably between the label and the text form field. Then information appears after the label at a logical point in the reading order.
+
+This also reduces the chance of the information overlapping with, for example, browser popups. For more information and user research, read the article [Avoid Messages Under Fields](https://adrianroselli.com/2017/01/avoid-messages-under-fields.html) by Adrian Roselli.
+
+Place for checkboxes and radio buttons the description between the legend and form fields.
+This is explained in page about [Descriptions in fieldsets]({{site.baseurl}}/docs/topics/forms/descriptions/fieldset-description/).
+
+## Examples
+
+{: .do }
+Place descriptions in a consistent location, preferably between the label and the form field.
+
+```html
+<label for="new-password">New password</label>
+<p id="description-new-password">
+    Choose a password of at least 8 characters long.
+</p>
+<input
+        id="new-password"
+        type="password"
+        name="new-password"
+        aria-describedby="description-new-password"
+        autocomplete="new-password"
+/>
+```
+
+{: .do }
+Place all descriptions, including error messages, in a consistent location, preferably between the label and the form field.
+New password
+
+```html
+<label for="name">New password</label>
+<div id="description-new-password">
+    Choose a password of at least 8 characters long.
+</div>
+<div id="error-wachtwoord">
+    Your chosen password is too short.
+</div>
+<input
+        id="new-password"
+        type="password"
+        name="new-password"
+        aria-describedby="description-new-password error-wachtwoord"
+        autocomplete="new-password"
+/>
+```
+
+
+{: .dont }
+Do not place the description below the form field.
+
+```html
+<!-- do not copy, this is not user-friendly -->
+<label for="new-password">New password</label>
+<input
+        id="new-password"
+        type="password"
+        name="new-password"
+        aria-describedby="description-new-password"
+        autocomplete="new-password"
+/>
+<p id="description-new-password">
+    Choose a password of at least 8 characters long.
+</p>
+```
+
+
+{: .dont }
+Do not place a description above the form field and an error message below it, be consistent in the location of descriptions.
```

Comment:
Break into two sentences: Do not place a description above the form field and an error message below it. Be consistent in the location of descriptions.